wemake_python_styleguide/options/defaults.py

```diff
@@ -143,6 +143,9 @@
 #: Maximum number of subjects in a ``match`` statement.
 MAX_MATCH_SUBJECTS: Final = 7  # 7 +- 0, guessed
 
+#: Maximum number of subjects in match statement.
+MAX_MATCH_CASES: Final = 6  # guessed
```

Comment:
```suggestion
MAX_MATCH_CASES: Final = 7  # guessed
```

I think that 6 is not enough :(

Reply:
Raised to 7 and fixed corresponding tests